Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -173,8 +172,12 @@ class BrowserViewController: UIViewController {
     }
 
     override var preferredStatusBarStyle: UIStatusBarStyle {
-        let isPrivate = tabManager.selectedTab?.isPrivate ?? false
-        return isPrivate ? UIStatusBarStyle.lightContent : UIStatusBarStyle.default
+        switch TabType.of(tabManager.selectedTab) {
```

Comment:
I think this should use the tab's theme, instead of the type.

Reply:
Good spot 👍